cishouseholds/pipeline/regex_testing.py

```diff
@@ -0,0 +1,307 @@
+
+other_hc = r"OPTICIAN|OPTHAL.*IST|ACCUP.*RE|THERAPIST|FOOT PRACTI(CIAN|TION)|HOMEOPATHY|PHYSCOT.*IST|\bMORT|HOSPITAL|\bNHS\b|MEDICAL EQUIP|TEACHING MEDICAL|WELLBEING PRACTITIONER"
+
+support_roles = "ASSISTANT"  # supporting types of people
+
+# patient facing
+patient_facing_negative_regex = match_with_exclusions(
+    r"ONLINE|ZOOM|MICROSOFT|MS TEAMS|SKYPE|GOOGLE HANGOUTS?|REMOTE|VIRTUAL|(ONLY|OVER THE) (TELE)?PHONE|((TELE)?PHONE|VIDEO) (CONSULT|CALL|WORK|SUPPORT)|(NO[TN]( CURRENTLY)?|NEVER) (IN PERSON|FACE TO FACE)|SH[EI]+LDING|WORK(ING)? (FROM|AT) HOME|HOME ?BASED|DELIVER(Y|ING)? PRESCRI",
+    r"(?<!NOT )OFFICE BASED",
+)
+patient_facing_positive_regex = "|".join(
+    [
+        r"PALLIATIVE CARE|(?<!NOT )PATI[EA]NT FACING|(LOOK(S|ING)? AFTER|SEES?|CAR(E|ING) (OF|FOR)) PATI[EA]NTS|(?<!NO )FACE TO FACE|(?<!NOT )FACE TO FACE",
+        r"(?<!NO )(DIRECT )?CONTACT WITH PATI[EA]NTS|CLIENTS COME TO (HER|HIS|THEIR) HOUSE",
+    ]
+)
+
+healthcare_classification = {
+    "Primary": [
+        "accident_and_emergency",
+        "additional_primary_hc",
+        "call_operator",
+        "dentist",
+        "general_practitioner",
+        "paramedic",
+        "pharmacist",
+    ],
+    "Secondary": [
+        "additional_secondary_hc",
+        "base_incluions",
+        "dietician",
+        "hc_theatre",
+        "hospital_generic",
+        "midwife",
+        "physiotherapist",
+    ],
+    "Other": ["hc_counsellor", "covid_test"],
+}
+
+social_care_classification = {
+    "Care/Residential home": ["residential_care"],
+    "Other": ["social_work", "child_care", "informal_care", "social_work"],
+}
+
+patient_facing_classification = {
+    "Y": [
+        "accident_and_emergency",
+        "additional_primary_hc",
+        "additional_secondary_hc",
+        "base_inclusions",
+        "dentist",
+        "dietician",
+        "doctor",
+        "general_practitioner",
+        "hc_counsellor",
+        "hc_theatre",
+        "hospital_generic",
+        "midwife",
+        "nurse",
+        "paramedic",
+        "pharmacist",
+        "physiotherapist",
+        "social_work",
+        "residential_care",
+    ],
+    "N": ["call_operator", "covid_test", "hc_admin", "hc_receptionist", "hc_secretary"],
+}
+
+healthcare_negative_roles = [
+    "transport",
+    "catering",
+    "teaching",
+    "media",
+    "retail",
+    "domestic",
+    "construction",
+    "religion",
+    "IT",
+    "public_service",
+    "vet",
+    "house_care",
+    "child_care",
+    "informal_care",
+    "residential_care",
+    "social_work",
+    "base_non_healthcare",
+]
+
+healthcare_positive_roles = [
```

Comment:
will this be used anywhere? In the creation of the binary health_care_classification?